python/cudf/cudf/core/dataframe.py

```diff
@@ -1755,6 +1755,18 @@ def _clean_renderable_dataframe(self, output):
         else:
             width = None
 
+        for col_name, col in output._data.items():
+            if isinstance(
+                col,
+                (
+                    cudf.core.column.timedelta.TimeDeltaColumn,
+                    cudf.core.column.datetime.DatetimeColumn,
+                ),
+            ):
+                output._data[col_name] = output._data[col_name].astype("str")
+            else:
```

Comment:
* Is this `else` bit a no-op? If so can we just omit it?
* Can we add a comment here indicating why we are converting datetimes/timedeltas to strings before handing off to Pandas?

Reply:
Done 👍